openapi/src/main/java/io/micronaut/openapi/visitor/ElementUtils.java

```diff
@@ -158,4 +167,33 @@ private static boolean findAnyAssignable(ClassElement type, List<String> typeNam
         }
         return false;
     }
+
+    /**
+     * Read classnames from JsonView annotation.
+     *
+     * @param element The element
+     *
+     * @return List of classnames from JsonView annotation.
+     */
+    @Nullable
+    public static List<String> getJsonViewClasses(@Nullable Element element) {
+        AnnotationValue<JsonView> classLvlJsonView = element != null ? element.getAnnotation(JsonView.class) : null;
+        if (classLvlJsonView == null) {
+            return null;
+        }
+        Map<CharSequence, Object> values = classLvlJsonView.getValues();
+        if (CollectionUtils.isEmpty(values)) {
+            return null;
+        }
+        AnnotationClassValue[] test = (AnnotationClassValue[]) values.get("value");
+        if (ArrayUtils.isEmpty(test)) {
+            return null;
+        }
+        List<String> jsonViewClasses = new ArrayList<>(test.length);
+        for (AnnotationClassValue value : test) {
+            jsonViewClasses.add(value.getName());
+        }
+
+        return jsonViewClasses;
```

Comment:
I believe you can just call `String[] classNames = element.getAnnotationMetadata().stringValues()`

Reply:
fixed